packages/@pollyjs/adapter/src/-private/interceptor.js

```diff
@@ -0,0 +1,35 @@
+const ABORT = Symbol();
+const PASSTHROUGH = Symbol();
+
+function setDefaults(interceptor) {
+  interceptor[ABORT] = false;
+  interceptor[PASSTHROUGH] = false;
+}
+
+export default class Interceptor {
+  constructor() {
+    setDefaults(this);
+  }
+
+  abort() {
```

Comment:
Might confuse people that this is aborting requests, i.e., https://developer.mozilla.org/en-US/docs/Web/API/AbortController/abort